yarn.lock

```diff
@@ -7722,7 +7717,7 @@ typedarray-to-buffer@^3.1.5:
   version "0.0.0"
   uid ""
 
-typescript@^4.3.4, typescript@^4.3.5, typescript@~4.3.5:
+typescript@4.3.5, typescript@^4.3.4, typescript@~4.3.5:
```

Comment:
Please revert this change

Reply:
this is temporary until https://github.com/rollup/plugins/issues/983 is fixed